os/mm/umm_heap/Make.defs

```diff
@@ -60,6 +60,10 @@ ifeq ($(CONFIG_BUILD_KERNEL),y)
 CSRCS += umm_sbrk.c
 endif
 
+ifeq ($(CONFIG_MM_KERNEL_HEAP),y)
+CSRCS += umm_heap_malloc.c
```

Comment:
How about another name like `assigned_malloc`?
`umm_malloc` also allocates the memory in the heap so that there is no difference between them in the naming point of view.
I suggest it because specialty of new function is assigned allocation to user heap.

Reply:
Changed to `malloc_user_at`. Thanks.